install.sh

```diff
@@ -1,7 +1,28 @@
 #!/bin/bash
 
-# Install operating system dependencies
-sudo apt-get install python2.7 python-pip python-django python-dev python-virtualenv libldap2-dev libsasl2-dev libssl-dev
+# Retrieve distribution and release version
+DISTRIBUTION=$(lsb_release -i 2> /dev/null | cut -d ':' -f 2 | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')
+RELEASE=$(lsb_release -r 2> /dev/null | cut -d ':' -f 2 | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')
+ERR_MSG="Unsupported Linux distribution, please install manually.
+See https://github.com/rosedu/wouso#installing-wouso for details.
+Aborting..."
+
+if [ -z "$DISTRIBUTION" -o -z "$RELEASE" ]; then
+    echo -e "$ERR_MSG"; exit 1
+fi
+
+# Install dependencies
+if [ "$DISTRIBUTION" == "Ubuntu" ]; then
+    sudo apt-get install python2.7 python-pip python-django python-dev python-virtualenv libldap2-dev libsasl2-dev libssl-dev
+elif [ "$DISTRIBUTION" == "Fedora" ]; then
+    if [ "$RELEASE" -ge 22 ]; then
+        sudo dnf -y install python-pip python-django python-devel python-virtualenv openldap-devel libgsasl-devel openssl-devel
+    else
+        sudo yum -y install python-pip python-django python-devel python-virtualenv openldap-devel libgsasl-devel openssl-devel
+    fi
+else
```

Comment:
Oh, right, I've added Mint in the readme but forgot to add it here too. Should be OK now.